admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

```diff
@@ -111,80 +111,89 @@
     <script type="text/javascript" src="${item}"></script>
 </sly>
 
-<sly data-sly-include="renderer.html"></sly>
+<sly data-sly-include="renderer.html" />
 <script>
     Vue.use(VueCodeMirror)
+    var $perAdminView = {
+        state: {},
+        adminPage: {},
+        adminPageStaged: {},
+        admin: {},
+        pageView: {
+            view: 'desktop'
+        },
+        nodes: {}
+    }
+
```

Comment:
this section was intentially formated as one line before to avoid the extra bytes in transport

Reply:
reverted